code/ui/generalhud/inspectmenu/InspectEntry.cs

```diff
@@ -0,0 +1,54 @@
+using Sandbox;
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+namespace TTTReborn.UI
+{
+    public class InspectEntry : Panel
+    {
+        public string ImagePath;
+        public string Description;
+
+        private readonly Image _inspectIcon;
+        private readonly Label _inspectQuickLabel;
+
+        public InspectEntry(Panel parent) : base(parent)
+        {
+            Parent = parent;
+
+            AddClass("rounded");
+            AddClass("text-shadow");
+            AddClass("background-color-secondary");
+
+            _inspectIcon = Add.Image();
+            _inspectIcon.AddClass("inspect-icon");
+
+            _inspectQuickLabel = Add.Label();
+            _inspectQuickLabel.AddClass("quick-label");
+        }
+
+        public void SetData(string imagePath, string description)
+        {
+            ImagePath = imagePath;
+            Description = description;
+
+            var icon = Texture.Load(imagePath);
+            if (icon == null)
+            {
+                icon = Texture.Load($"/ui/none.png");
```

Comment:
One liner (null-coalescing-operator). Could be included into the panelbackground declaration below as well.